packages/migrate/src/MigrateEngine.ts

```diff
@@ -135,11 +142,42 @@ export class MigrateEngine {
     return this.runCommand(this.getRPCPayload('evaluateDataLoss', args))
   }
 
+  public getDatabaseDescription(schema: string): Promise<string> {
+    return this.runCommand(this.getRPCPayload('getDatabaseDescription', { schema }))
+  }
+
   /**
    * Get the database version for error reporting.
+   * - TODO: this command currently requires the engine to be initialized with a schema path.
+   *   The IntrospectionEngine supported reading a database version from an inline schema or URL instead,
+   *   which if more flexible and fundamental for the error reporting use case. We should add that here too.
```

Comment:
Can we already create an issue for this?

Reply:
There's already an issue linked in the comments

Reply:
That seemed to be a separate TODO, so I expected a different issue. 
(The other issue is also not really about that TODO, but a requesite for that TODO)